libs/api/domains/intellectual-properties/src/lib/mapper.ts

```diff
@@ -28,3 +36,19 @@ export const mapTrademarkSubtype = (
     ? TrademarkSubType.CERTIFICATION_MARK
     : TrademarkSubType.TRADEMARK
 }
+
+export const mapFullAddress = (
+  streetAddress?: string | undefined | null,
+  postalCode?: string | undefined | null,
+  city?: string | undefined | null,
+) => {
+  if (!streetAddress) {
+    return `${postalCode + ' '}${city}`
+  }
+
+  if (!postalCode && !city) {
+    return streetAddress
+  }
```

Comment:
Can we be sure ``${postalCode + ' '}${city}` is available when no street address and `streetAddress` is available when no postalcode and city?